src/ShortestPaths/bfs.jl

```diff
@@ -26,7 +26,8 @@ struct BFS{T} <: ShortestPathAlgorithm
     maxdist::Int64
 end
 
-BFS(; sort_alg=Traversals.NOOPSort, neighborfn=outneighbors, maxdist=typemax(Int64)) = BFS(Traversals.BreadthFirst(sort_alg=sort_alg, neighborfn=neighborfn), maxdist)
+BFS(; traversal=Traversals.BreadthFirst(), maxdist=typemax(Int64)) = BFS(traversal, maxdist)
```

Comment:
This is a bit confusing, why can a BFS search use some other traversal function than `BreadthFirst`?

Reply:
It can use `ThreadedBreadthFirst`, for example.